src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries.mdx

```diff
@@ -8,31 +8,65 @@ redirects:
 
-When you run a [NRQL query](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-is-nrql), it will display the number of events inspected, as shown below:
+* Limit the number complex queries (for example, queries with `FACET` or `TIMESERIES` clauses, or queries of over a million events) that run at the same time.
+* Limit the number of queries running concurrently over extended periods of time to a maximum of five, especially if they include complex queries.
+
+## NRQL query limits [#query-limits]
+
+There are a few different NRQL query limits: 
+
+* Number of queries an account can make in a given time range
+* Number of data points an account can query (inspect) in a given time range
+* How long a query can run for before being considered an error and stopping 
+
+We'll go into more detail on these. 
+
+### Limits on number of queries [#query-count-limits]
+
+The limit on NRQL queries is 3,000 queries per account per minute. When this limit is exceeded, queries may be rejected until the number of queries sent per minute no longer exceeds the limit. 
+
+### Limits on number of data points inspected [#inspected-count-limits]
+
+When you run a NRQL query, it displays the number of data points inspected, as shown below:
 
 <img
   title="New Relic inspected event count"
   alt="New Relic inspected event count"
   src={eventCountModal}
 />
 
-In this context, "events" is used in a general sense to refer to all [NRQL-available objects](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-you-can-query); this includes events, metrics, logs, and distributed tracing (span) data.
+In this context, the "events" term is used in a general sense to refer to all [NRQL-available data points](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-you-can-query) that have been inspected in a query.
 
-Each New Relic account has limits on the total number of events that can be inspected. There are limits that apply over two different time frames:
+Every New Relic account has limits on the total number of data points that can be inspected in a given time range. These limits vary depending on which [data plan](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you have: 
 
-* A rolling 30-minute time window
-* A 24-hour period
+<CollapserGroup>
+  <Collapser
+    id="data-plus"
+    title="Data Plus"
+  >
 
-These limits are as follows:
+The [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) data plan includes these query limits: 
+
+* Inspected data points per minute: 100 billion
+* Inspected data points per 30 minutes: one trillion
+
+?? SHOULD WE MENTION SOMETHING ABOUT 24 HOUR LIMITS, AS WE SAY FOR ORIGINAL DATA PLAN BELOW? SHOULD WE MENTION SOMETHING ABOUT THE DISCREPANCY BETWEEN THE 100B AND 1T, AS THAT MATH IS WEIRD? ?? 
```

Comment:
Calling this out so you know to update/remove it! We talked about this briefly in our meeting last week.